scene/animation/animation_tree.cpp

```diff
@@ -991,6 +1001,7 @@ void AnimationTree::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::NODE_PATH, "anim_player", PROPERTY_HINT_NODE_PATH_VALID_TYPES, "AnimationPlayer"), "set_animation_player", "get_animation_player");
 
 	ADD_SIGNAL(MethodInfo(SNAME("animation_player_changed")));
+	ADD_SIGNAL(MethodInfo(SceneStringName(node_notification), PropertyInfo(Variant::STRING_NAME, "node"), PropertyInfo(Variant::INT, "what")));
```

Comment:
```suggestion
	ADD_SIGNAL(MethodInfo(SceneStringName(animation_node_notification), PropertyInfo(Variant::STRING_NAME, "animation_node_path"), PropertyInfo(Variant::INT, "what")));
```
"node" should be avoided because it is confusing with Node/Node2D/Node3D classes.